docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -188,3 +188,51 @@ FETCH FIRST N ROWS``.
 
 Implementation and support is connector-specific since different data sources
 support different SQL syntax and processing.
+
+
+Following is presented a concrete example of a Top-N pushdown query
+on top of a PostgreSQL database::
+
+    SELECT id, name
+    FROM postgresql.public.company
+    ORDER BY id
+    LIMIT 5;
+
+You can get the explain plan by prepending the above query with ``EXPLAIN``::
+
+    EXPLAIN SELECT id, name
+    FROM postgresql.public.company
+    ORDER BY id
+    LIMIT 5;
+
+Following is presented the query plan of the ``EXPLAIN`` query.
+
+.. code-block:: text
+
+ Fragment 0 [SINGLE]
+     Output layout: [id, name]
+     Output partitioning: SINGLE []
+     Stage Execution Strategy: UNGROUPED_EXECUTION
+     Output[id, name]
+     │   Layout: [id:integer, name:varchar]
+     │   Estimates: {rows: ? (?), cpu: ?, memory: 0B, network: ?}
+     └─ RemoteSource[1]
+            Layout: [id:integer, name:varchar]
+
+ Fragment 1 [SOURCE]
+     Output layout: [id, name]
+     Output partitioning: SINGLE []
+     Stage Execution Strategy: UNGROUPED_EXECUTION
+     TableScan[postgresql:public.company public.company sortOrder=[id:integer:int4 ASC NULLS LAST] limit=5, grouped = false]
+         Layout: [id:integer, name:varchar]
+         Estimates: {rows: ? (?), cpu: ?, memory: 0B, network: 0B}
+         name := name:varchar:text
+         id := id:integer:int4
+
+Note within ``SOURCE`` fragment that the ``TableScan`` element
```

Comment:
This depends on the connector though. The string comes from how the connector implemented `ConnectorTableHandle#toString`.

The actual thing which lets you know pushdown happenned is the fact that there's no TopNNode or LimitNode in the plan - only a TableScan.

@mosabua can point to some other pushdown docs which can be consulted for the best way to word it. Maybe https://github.com/trinodb/trino/pull/8463/files#diff-3a35320d12c22f7d413cda275bad87d6467ea3482f35eda3f89c51af38c99e85R163-R165?